### ISSUE TITLE
[Bug] "Earned Achievements" requires custom variable for progress bar

### ISSUE THREAD

**Issue by Naval-777** [Lacro59/playnite-successstory-plugin#219]
The "Earned Achievements" progress bar found in the "Games Details" tab in fullscreen mode shows the progress % out of 100 instead of the amount of achievements that the game contains. My two examples of this are shown on Civilization 5 (34 out of 286 unlocked, however progress bar appears to be 34% across) and The Binding of Isaac: Rebirth (171 out of 637 unlocked, progress bar is stretched more than other games to accommodate for a number above 100, this effect also happens on Mass Effect: LE with all 127 achievements unlocked). 

Theme = PlayniteModernUI

![civilization5bug](https://user-images.githubusercontent.com/94616409/142416030-8274e938-ab23-439a-b28d-1cb4f09d7b49.PNG)
![isaacbug](https://user-images.githubusercontent.com/94616409/142416071-c6690e6f-9bb1-45c0-9bab-a38d2872446c.PNG)
![masseffectbug](https://user-images.githubusercontent.com/94616409/142416081-a0037f62-59fd-4197-81f0-a7b627272d43.PNG)

**Comment by Lacro59** [comment]
For me, it's an issue on the them here: https://github.com/davidkgriggs/PlayniteModernUI/blob/9bee29dc71245501710e38d5a28bedc4d1db3a2a/source/Views/GameDetails.xaml#L312

put 
`Width="{PluginSettings Plugin=SuccessStory, Path=Total}"`
instead of
`Width="100"`

**Comment by Naval-777** [comment]
I just tried using the code in your reply, however this just seems to have increased the size of the progress bar to the maximum size at the same ratio instead of asking the progress bar to increase in individual intervals until reaching the maximum. I made sure to use the same line as highlighted in the link. Here is an example of the result:

![uhohnotgood](https://user-images.githubusercontent.com/94616409/142715337-8c6a9396-e95e-40c6-8159-a3ba79036977.PNG)

Should I also post this issue to the ModernUI GitHub page so they can patch it in an official release?
